pkg/controller/environment.go

```diff
@@ -364,7 +373,8 @@ func (env *K8sEnvironment) PrepareRun(stopCh <-chan struct{}) error {
 		cont.namespaceInformer.HasSynced,
 		cont.replicaSetInformer.HasSynced,
 		cont.deploymentInformer.HasSynced,
-		cont.podInformer.HasSynced)
+		cont.podInformer.HasSynced,
+		cont.oobPolicyInformer.HasSynced)
```

Comment:
This is an optional CR, shoudl be implemented as a CRD hook.. Sorry I missed this earlier.